src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,141 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
0 is the default value